src/test/CMakeLists.txt

```diff
@@ -25,13 +25,20 @@ target_sources_local(nb PRIVATE
     tutorial_fluctuating_v2.cpp
     tutorial_mustache.cpp
     tutorial_render_simple.cpp    
-    tutorial_slow_v1.cpp
```

Comment:
Workaround for BM build issue - sleep/usleep missing, required sleep_for used in these tests.